compiler-core/src/error.rs

```diff
@@ -1224,6 +1226,13 @@ Hint: Add some type annotations and try again."
                         text.push_str(field);
                     }
 
+                    match variants {
+                        RecordVariants::HasVariants => {
+                            text.push_str("\n\nNote: The field you are trying to access might not be consistently present or positioned across the custom type's variants, preventing reliable access. Ensure the field exists in the same position and has the same type in all variants to enable direct accessor syntax.");
```

Comment:
Wrap this long string please! There's a `wrap` function which is useful for this. You can see it used below.

Reply:
Done. I updated the branch from `gleam-lang:main` wrapped the msg and also separated the `\n\n` from the message itself.
Thanks for the tip.